src/styles/hc-2018/ternary.styl

```diff
@@ -208,7 +208,7 @@ a::after {
     color: var(--primary-color);
 }
 a[rel="external"]::after {
-    content:"";
+    content:" →";
```

Comment:
The spacing between some of the icons on the splash page, particularly the square ones, looks a bit off to me.

Reply:
Spacing between all these icons uses the predefined `--gutter` measurement (but because Twitter isn't square it has extra spacing)

Reply:
If you're referring to the space between the icon and the arrow, that _has_ been fixed with https://github.com/hackcambridge/hack-cambridge-website/pull/261/commits/b64bdad342703a2a4736dc46ede287efef39cbb0